prac_05/hex_colours.py

```diff
@@ -0,0 +1,13 @@
+"""
+Create a program that allows you to look up hexadecimal colour codes.
+"""
+
+COLOUR_CODES = {"AliceBlue": "#f0f8ff", "AntiqueWhite": "#faebd7", "AntiqueWhite1": "#ffefdb", "cyan1": "#00ffff",
+                "DarkGoldenrod3": "#cd950c", "aquamarine1": "#7fffd4", "DeepPink1": "#ff1493", "gold1": "#ffd700",
+                "gray": "#bebebe", "pale": "#db7093"}
+
+
+colour_name = input("Enter a colour name: ")
+while colour_name != "":
+    print("The code for \"{}\": {}".format(colour_name, COLOUR_CODES.get(colour_name)))
```

Comment:
You could possibly store your colour codes as all lower case (or upper) and use .lower() to allow the user to not have to directly follow the way capitals are used. I might wonder why I need to enter AntiqueWhite1 for one but cyan1 for the other if I didn't already know the colours saved in the program

Reply:
Thank you for your helpful comments, Samuel. That is what I'm also confused about. If I put `.lower`, the input colour_name does not match COLOUR_CODES. e.g.) if a user enters `AliceBlue` as input, then the program converts it to `aliceblue` (by using .lower method), but it does not match `AliceBlue` in COLOUR_CODES dictionary.

Reply:
The easiest fix would be to just save the names as all lowercase in the COLOUR_CODES dictionary and just use the .lower() on the user input.  This will mean that when you print out the code it will be all lowercase so this might not fall in line with exactly what the question wants from the program. I think your program has addressed the question anyway and this is just a minor convenience thing.

Reply:
That is such a great idea! I Thank you very much.